### ISSUE TITLE
change / delete surround delimiters stopped working with evil-cp-change / evil-cp-delete

### ISSUE THREAD

**Issue by practicalli-john** [emacs-evil/evil-surround#204]
A change on or after 27th June 2023 to the evil-surround package causes change surround and delete surround to fail with the following message

```
evil-motion-range: No surrounding delimiters found
```

I'm assuming either evil-surround has change or evil itself has change the way it calls `evil-cp-change` or `evil-cp-delete.

Both commands work with a Melpa package build `evil-surround-20230627.2052` containing the `evil-surround.el` file with a timestamp of 17 August 2023 at 11.23 UTC

First discovered in Spacemacs and reported here: https://github.com/syl20bnr/spacemacs/issues/16144

**Comment by tomdl89** [comment]
Thanks for the bug report @practicalli-john - I'll take a look as soon as I have some time.

**Comment by practicalli-john** [comment]
All looks good now to me.  

I updated the recipe to use the latest commit and restarted Emacs.  I assume Melpa will pick up the new commits in the current build that should finish in the next two hours and the recipe will not longer be required.

```elisp
(evil-surround
  :location
  (recipe :fetcher github
          :repo "emacs-evil/evil-surround"
          :commit "0d860be74165ceb8314742e4191cdad693f40a6d"))
```

**Comment by tomdl89** [comment]
The "over-deletion" problem is fixed in bd98f7f5492446885dc4c6b0a7579c75c22ac829
